### PR TITLE
18886: Adds 'missing_value_accuracy' in conditioned prediction stats, MINOR

### PR DESCRIPTION
**Pull request by cademack**
Adds logic to compute 'missing_value_accracy' in get_prediction_stats when a condition is specified.

### CODE REVIEW

File: trainee_template/prediction_stats.amlg [hunk 1]
```diff
@@ -363,6 +364,58 @@
 			)
 		)
 
+		(if (contains_value stats "missing_value_accuracy")
+			(seq
+				(declare (assoc
```
Comment (howsoRes) [inline trainee_template/prediction_stats.amlg]: seq-declare -> let statement

File: trainee_template/prediction_stats.amlg [hunk 2]
```diff
@@ -363,6 +364,58 @@
...
+							)
+							features_map
+						)
+				))
+
+				;filter out the features that did not have any nulls
+				(assign (assoc
+					null_feature_cases_map
+						(filter
+							(lambda (> (size (current_value)) 0) )
```
Comment (howsoRes) [inline trainee_template/prediction_stats.amlg]: simplify to (size (current_value))

File: trainee_template/prediction_stats.amlg [hunk 3]
```diff
@@ -363,6 +364,64 @@
...
+					feature_residuals_lists (null)
+					robust_residuals robust
+				)
+
+				(declare (assoc
+					null_feature_cases_map
+						(map
+							(lambda
+								(filter
+									(lambda (= (retrieve_from_entity (current_value) (current_index 1)) (null)) )
```
Comment (howsoRes) [inline trainee_template/prediction_stats.amlg]: i suggest you replace this filter with:
(contained_entities (list 
  (query_in_entity_list case_ids)
  (query_equals (current_value 1) (null) )
))

File: trainee_template/prediction_stats.amlg [hunk 4]
```diff
@@ -363,6 +364,64 @@
...
+			(let
+				(assoc
+					features_map (zip features)
+					null_prediction_map (null)
+					feature_residuals_lists (null)
+					robust_residuals robust
+				)
+
+				(declare (assoc
+					null_feature_cases_map
```
Comment (howsoRes) [inline trainee_template/prediction_stats.amlg:378]: add a comment as to what this map holds

Comment (cademack) [reply trainee_template/prediction_stats.amlg:378]: will do

File: trainee_template/prediction_stats.amlg (hunk 3 above)
Comment (cademack) [reply trainee_template/prediction_stats.amlg]: done

**Review by howsoRes — APPROVED** [review]

**Pull request merged**